beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/filter/EventTypeTableListenerEventFilter.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.scheduler.apiary.filter;
+
+import com.expedia.apiary.extensions.receiver.common.event.EventType;
+import com.expedia.apiary.extensions.receiver.common.event.ListenerEvent;
+import org.springframework.stereotype.Component;
+import static com.expediagroup.beekeeper.core.model.LifeCycleEventType.UNREFERENCED;
+import static com.expediagroup.beekeeper.core.model.LifeCycleEventType.EXPIRED;
+
+import java.util.Map;
+
+@Component
+public class EventTypeTableListenerEventFilter implements ListenerEventFilter {
+
+  @Override
+  public boolean filter(ListenerEvent listenerEvent) {
+    if (listenerEvent == null) {
```

Comment:
I think this null check is now done in the `FilteringMessageReader` so can be removed